### PR TITLE
revised account management part

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 - [Overview](home)
 - [Getting Started](quickstart)
 - [Onboarding Public Officers using non-SE Device](onboard-public-officers-using-non-se-machines)
-  - [Log in to TechPass Portal](log-into-techpass-portal-po-non-se)
+- Manage security verification
+  - [Set up security verification for WOG accounts](manage-security-verification-for-wog-account)
```
Comment (yilunyou) [inline docs/_sidebar.md:5]: Suggestion:
[Set up security verification for WOG account] < remove s from accounts
[Reset security verification for WOG account] < Re set up seems abit funny )

File: docs/_sidebar.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+  - [Re set up security verification for WOG account](re-set-up-security-verification-for-wog-account)
+- [Log in to TechPass Portal](log-into-techpass-portal-po-non-se)
```
Comment (yilunyou) [inline docs/_sidebar.md:7]: qns ah.. looking at the sidebar item. 
[Log in to TechPass Portal] < seems to cater to everyone . both po and vendors, etc.
(log-into-techpass-portal-po-non-se) < but this md file seems to named for jus non-se po. is that the intention?
if yes, then u might wan to rename sidebar item. if not, rename the md file

File: docs/manage-security-verification-for-wog-account.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Set up security verification for Whole of Government (WOG) account
+
+We know that public officers can use their TechPass account to access Singapore Tech Stack (SGTS) services and tools. However, if public officers want to access SGTS services and tools from their Government Managed Device (GMD) using TechPass, they must first set up security verification for their WOG account. This adds an additional level of security layer when they access these resources from their GMD.
```
Comment (yilunyou) [inline docs/manage-security-verification-for-wog-account.md:3]: Suggestion: 
"Public officers can use their TechPass account to access Singapore Tech Stack (SGTS) services and tools across Non-SE GSIB and/or GMD devices. ........... This additional level of security layer is required when they access these resources from their GMD"

File: docs/manage-security-verification-for-wog-account.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+To achieve this, public officers set up a multi-factor authentication (MFA) for their WOG account using their Government Standard Imaging Build (GSIB) device.
```
Comment (yilunyou) [inline docs/manage-security-verification-for-wog-account.md:5]: "To achieve this, public officers need to set up a mul"

File: docs/re-set-up-security-verification-for-wog-account.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+_To re set up security verification for WOG account:_
+
+1. From your GSIB device, go to [Microsoft Additional security verification](https://account.activedirectory.windowsazure.com/proofup.aspx). If prompted, sign in to your WOG account.
+<kbd>![delete-old-device](assets/images/security-verification-for-wog/re-setup-wog-mfa/delete-old-device.png)</kbd>
+2. Click **Delete** beside the device that you want to remove.
+<kbd>![deletion-in-progress](assets/images/security-verification-for-wog/re-setup-wog-mfa/deletion-in-progress.png)</kbd>
+
+3. After it is removed, go to your Authenticator app on your old mobile phone or device and remove the existing MFA of your WOG account(SG GovtM365).
+
+?> This only removes the authentication method for the WOG account in that device.
```
Comment (yilunyou) [inline docs/re-set-up-security-verification-for-wog-account.md:21]: suggestion: 
"This only removes the MFA profile for WOG account in that device."

File: docs/_sidebar.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/_sidebar.md:5]: :-), got it overlooked this!

File: docs/re-set-up-security-verification-for-wog-account.md (hunk 5 above)
Comment (SenthilKumarSivakami) [reply docs/re-set-up-security-verification-for-wog-account.md:21]: done.

File: docs/manage-security-verification-for-wog-account.md (hunk 4 above)
Comment (SenthilKumarSivakami) [reply docs/manage-security-verification-for-wog-account.md:5]: done

File: docs/manage-security-verification-for-wog-account.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply docs/manage-security-verification-for-wog-account.md:3]: Done.

File: docs/_sidebar.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/_sidebar.md]: renamed the file.

**Pull request merged**